test/using_witnet.js

```diff
@@ -96,11 +97,11 @@ contract("UsingWitnet", accounts => {
       assert.equal(drReward, reward * 2)
     })
 
-    it("requester balance should decrease after rewards upgrade", async () => {
-      const afterBalance = await web3.eth.getBalance(accounts[1])
-      assert(afterBalance < lastAccount0Balance - reward)
-      lastAccount0Balance = afterBalance
-    })
+    // it("requester balance should decrease after rewards upgrade", async () => {
+    //   const afterBalance = await web3.eth.getBalance(accounts[1])
+    //   assert(afterBalance < lastAccount0Balance - reward)
+    //   lastAccount0Balance = afterBalance
+    // })
```

Comment:
@clbartoli and I fixed the tests for `using_witnet.js`.

You can cherry pick our commit from my repo under the same branch name: https://github.com/mariocao/witnet-ethereum-bridge/commit/3e8c883590568e4afe10b5000c0f43cdf063086e

Reply:
Done.